Dockerfile

```diff
@@ -1,13 +1,15 @@
-FROM openjdk:8-jre
+FROM openjdk:8-jre-alpine
 
-ENV CEREBRO_VERSION 0.7.3
-RUN cd /opt/ \
-    && wget -O cerebro-${CEREBRO_VERSION}.tgz https://github.com/lmenezes/cerebro/releases/download/v${CEREBRO_VERSION}/cerebro-${CEREBRO_VERSION}.tgz \
-    && tar zxvf cerebro-${CEREBRO_VERSION}.tgz \
-    && rm cerebro-${CEREBRO_VERSION}.tgz \
-    && mkdir cerebro-${CEREBRO_VERSION}/logs \
-    && mv cerebro-${CEREBRO_VERSION} cerebro
+RUN apk add bash
+
+ENV CEREBRO_VERSION 0.8.0
+ADD ./cerebro-${CEREBRO_VERSION}.tgz /opt
```

Comment:
Does this mean that we need a local file named cerebro-${CEREBRO_VERSION}.tgz before building the image? If so, won't it break the automatic process of docker hub?